adafruit_ht16k33/segments.py

```diff
@@ -168,17 +170,16 @@ def __init__(
         self._chars = chars_per_display * len(self.i2c_device)
         self._bytes_per_char = 2
 
-    def print(self, value: Union[str, int, float], decimal: int = 0) -> None:
+    def print(self, value: Union[str, float], decimal: int = 0) -> None:
```

Comment:
Int is a valid value type for `print()`. It is just passed to `_number()`

Reply:
`Union[float, int]` is equivalent to `float` according to [PEP 484](https://peps.python.org/pep-0484/#the-numeric-tower)